### DISCUSSION TITLE
希望增加「听写模块」

### DISCUSSION THREAD

**Discussion by freeyob** [ZuodaoTech/everyone-can-use-english#1149] [Ideas]
我想跟读之前先试着听写，训练自己对音素的识别能力。不知道大家有没有类似需求。

**Comment by xiaolai** [comment]
真的是伪需求…… 听写实际上效率挺低的，听不懂，根本写不出来。听得懂，又何必听写？

从正面老老实实练，没必要还没怎样，先反过来练。

加油！
